docs/09_firebase_deployment.md

```diff
@@ -0,0 +1,201 @@
+# Metrics firebase deployment
+
+> Summary of the proposed change
+
+Describe the process of deployment of the Metrics application to the Firebase Hosting and deploying the Cloud functions needed to create test data for the application.
+
+
+# References
+
+> Link to supporting documentation, GitHub tickets, etc.
+
+- [Getting started with Firestore](https://cloud.google.com/firestore/docs/client/get-firebase)
+- [Deploying to Firebase Hosting](https://firebaseopensource.com/projects/firebase/emberfire/docs/guide/deploying-to-firebase-hosting.md/)
+- [Getting started with Firebase Cloud Functions](https://firebase.google.com/docs/functions/get-started)
+
+# Motivation
+
+> What problem is this project solving?
+
+The Metrics application instance with test data accessible via a link.
```

Comment:
Do we need to mention test data here? As it could be an optional step for real deployment?